apps/backend/src/data/migrations/1654702006571-create-menu-table.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey
+} from 'typeorm';
+
+export class CreateMenuTable1654702006571 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'menu',
+        columns: [
+          {
+            name: 'id',
+            type: 'int',
+            isGenerated: true,
+            generationStrategy: 'increment',
+            isPrimary: true
+          },
+          {
+            name: 'store_id',
```

Comment:
specify for each column if it can be or cannot be null. For the possible options for a column can be found inside the typeorm column `node_modules/typeorm/schema-builder/options/TableColumnOptions.d.ts`